RELEASE_NOTES.rst

```diff
@@ -18,6 +18,7 @@ Highlights:
    - Improved documentation
    - Expose the DAC coarse mixer and ``sif_sync``
    - Exposes upconverter calibration and enabling/disabling of upconverter LO & RF outputs.
+   - Add helpers to align Phaser updates to the RTIO timeline (``get_next_frame_timestamp()``)
```

Comment:
```suggestion
   - Add helpers to align Phaser updates to the RTIO timeline (``get_next_frame_mu()``)
```

Reply:
Sorry, forgot that one. Fixed in 4d7bd3e.